docs/xxidletuningminidlewaittime.md

```diff
@@ -26,7 +26,7 @@
 
 ** (Linux&reg; only) **
 
-When the OpenJ9 VM is idle, this option controls the minimum length of time that the VM must be idle before the status of the VM is set to idle. Further tuning options control the compaction of the object heap and the release of free memory pages, which reduces the footprint of the VM.
+The OpenJ9 VM can automatically detect when it is running inside a container. When the OpenJ9 VM is idle, this option controls the minimum length of time that the VM must be idle before the state of the VM is set to idle. When the state changes to idle, a garbage collection cycle runs, the object heap is compacted, and free memory pages are released back to the operating system, which reduces the footprint of the VM. Garbage collection and compaction are controlled by the `-XX:+IdleTuningGcOnIdle` and `-XX:+IdleTuningCompactOnIdle` options, which are enabled by default when the OpenJ9 VM is running inside a container.
```

Comment:
I think the description is good, but the first statement is not really needed, as container detection is not related to this option.